Automattic/wp-calypso
# JP Manage: Add a card with Jetpack Products to the Overview page

Resolve: https://github.com/Automattic/jetpack-manage/issues/147 https://github.com/Automattic/jetpack-manage/issues/148

## Proposed Changes

This PR adds a card to the Overview page with a sample of some available Jetpack products for the Agency. 

- Technical scope: pf36In-mf-p2#tools-component
-  Design: pf36In-fY-p2

![image](https://github.com/Automattic/wp-calypso/assets/9832440/202e87ff-9237-421e-acc9-b957e3a8e2eb)

## Testing Instructions

- Review de code
- Check the design and ensure that the products in the design matches the ones in this PR. If you don't see them, probably your account should be updated/migrated (ask Atanas)
- When you click on `View All` it should take you to the `/partner-portal/issue-license` page
- Each product has a link `More about...` for now it will take you to the marketing page of the product on jetpack.com. We have to implement the Lightbox inside of the Overview page, to be able to show the product info in the Partner context. It will addressed in a follow-up PR.
- Check that the link triggers the track events:
   - View All => `calypso_jetpack_manage_overview_products_view_all_click`
   - More about... => `calypso_jetpack_manage_overview_products_more_about_click` with a the property `product`

## Pre-merge Checklist
